src/generate_applications.jl

```diff
@@ -4,8 +4,8 @@ include("meta.jl")
 
 using Polymake.Meta
 
-json_dir = joinpath(@__DIR__, "..", "deps", "json")
 generated_dir = joinpath(@__DIR__, "generated")
+json_dir = joinpath(generated_dir, "json")
```

Comment:
aren't these already defined in `src/Polymake.jl`?

Reply:
yes, removed